terraform/modules/hub/files/tasks/frontend_xlarge.json

```diff
@@ -0,0 +1,171 @@
+    }, {
+      "Name": "COUNTRY_DISPLAY_LOCALES",
+      "Value": "/verify-frontend/federation/configuration/display-locales/countries"
+    }, {
+      "Name": "EIDAS_SCHEMES_DIRECTORY",
+      "Value": "/verify-frontend/federation/configuration/eidas/schemes"
+    }, {
+      "Name": "ZDD_LATCH",
+      "Value": ".service_unavailable"
+    }, {
+      "Name": "POLLING_WAIT_TIME",
+      "Value": "2"
+    }, {
+      "Name": "METRICS_ENABLED",
+      "Value": "false"
+    }, {
+      "Name": "LOG_LEVEL",
+      "Value": "${log_level}"
+    }, {
+      "Name": "RULES_DIRECTORY",
+      "Value": "/verify-frontend/federation/configuration/idp-rules"
+    }, {
+      "Name": "RULES_DIRECTORY_VARIANT",
+      "Value": "/verify-frontend/federation/configuration/idp-rules-variant"
+    }, {
+      "Name": "RULES_C_DIRECTORY",
+      "Value": "/verify-frontend/federation/configuration/idp-rules-variant-c"
+    }, {
+      "Name": "RP_CONFIG",
+      "Value": "/verify-frontend/federation/configuration/relying_parties.yml"
+    }, {
+      "Name": "IDP_CONFIG",
+      "Value": "/verify-frontend/federation/configuration/identity_providers.yml"
+    }, {
+      "Name": "CYCLE_THREE_ATTRIBUTES_DIRECTORY",
+      "Value": "/verify-frontend/federation/configuration/cycle-three-attributes"
+    }, {
+      "Name": "AB_TEST_FILE",
+      "Value": "/verify-frontend/federation/configuration/ab_test/${ab_test_file}"
+    }, {
+      "Name": "ABC_VARIANTS_CONFIG",
+      "Value": "/verify-frontend/federation/configuration/special-cases/abc-variants.yml"
+    }, {
+      "Name": "SEGMENT_DEFINITIONS",
+      "Value": "/verify-frontend/federation/configuration/segment_definitions.yml"
+    }, {
+      "Name": "SEGMENT_DEFINITIONS_VARIANT_C",
+      "Value": "/verify-frontend/federation/configuration/segment_definitions_variant_c.yml"
+    }, {
+      "Name": "PUBLIC_PIWIK_HOST",
+      "Value": "https://www.${domain}:443/analytics"
+    }, {
+      "Name": "INTERNAL_PIWIK_HOST",
+      "Value": "${analytics_endpoint}:443/piwik.php"
+    }, {
+      "Name": "PIWIK_SITE_ID",
+      "Value": "${matomo_site_id}"
+    }, {
+      "Name": "ZENDESK_URL",
+      "Value": "${zendesk_url}"
+    }, {
+      "Name": "ZENDESK_USERNAME",
+      "Value": "${zendesk_username}"
+    }, {
+      "Name": "SAML_PROXY_HOST",
+      "Value": "https://saml-proxy.${domain}:443"
+    }, {
+      "Name": "VERIFY_PRODUCT_PAGE",
+      "Value": "https://govuk-verify.cloudapps.digital/"
+    }, {
+      "Name": "RAILS_ENV",
+      "Value": "production"
+    }, {
+      "Name": "RAILS_LOG_TO_STDOUT",
+      "Value": "true"
+    }, {
+      "Name": "CROSS_GOV_GOOGLE_ANALYTICS_TRACKER_ID",
+      "Value": "${cross_gov_ga_tracker_id}"
+    }, {
+      "Name": "CROSS_GOV_GOOGLE_ANALYTICS_DOMAIN_LIST",
+      "Value": "${cross_gov_ga_domain_names}"
+    }, {
+      "Name": "PUBLISH_HUB_CONFIG_ENABLED",
+      "Value": "${publish_hub_config_enabled}"
```

Comment:
this has been updated on master, could you update it as per https://github.com/alphagov/verify-infrastructure/pull/381 ?

Reply:
This has been done.

Reply:
NOt yet. Let me update the new file.